package.json

```diff
@@ -53,6 +53,7 @@
       "editorconfig-checker -exclude \".*[\\\\/]templates[\\\\/].*\" -ignore-defaults --verbose"
     ]
   },
+  "homepage": "https://amadeusitgroup.github.io/otter/",
```

Comment:
Don't we want to target `https://amadeusitgroup.github.io/otter/#/localization` for the localization package for example ?